brapi_to_isa.py

```diff
@@ -389,7 +389,7 @@ def get_trials( brapi_client : BrapiClient):
         TRIAL_IDS = []
         for my_study_id in STUDY_IDS:
             my_study = brapi_client.get_study(my_study_id)
-            TRIAL_IDS += my_study["trialDbIds"]
+            TRIAL_IDS += my_study["trialDbId"]
```

Comment:
We have another case of v1.1 vs 1.2 problem here.
Something like : 
```suggestion
           if my_study["trialDbId"] is not None:
              TRIAL_IDS += my_study["trialDbId"]
            else if my_study["trialDbIds"] is not None:
               TRIAL_IDS += my_study["trialDbIds"]
```

Reply:
I was not aware of the differences between the two versions. Good idea I will change it tomorrow